From 7de794ef3b0edcdf7dd0bee43cecef71c59cfead Mon Sep 17 00:00:00 2001
From: Sanjay Kumar Champati <csanjay@mistralsolutions.com>
Date: Fri, 10 Dec 2010 10:32:45 -0800
Subject: [PATCH 5/5] Modified defconfig for WL1271 DC support on AM1808

Modified the defconfig file to support WL1271 DC configuration
on AM1808. This defconfig file helps to configure WLAN and BT
for WL1271 on AM1808 platform.

Signed-off-by: Sanjay Kumar Champati <csanjay@mistralsolutions.com>
---
 arch/arm/configs/da850_omapl138_defconfig |   40 ++++++++++++++++++++++++++--
 1 files changed, 37 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/da850_omapl138_defconfig b/arch/arm/configs/da850_omapl138_defconfig
index b312050..ec374b0 100644
--- a/arch/arm/configs/da850_omapl138_defconfig
+++ b/arch/arm/configs/da850_omapl138_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.33-rc4
-# Thu Feb 18 17:57:55 2010
+# Wed Dec  1 04:52:46 2010
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -243,6 +243,7 @@ CONFIG_ARCH_DAVINCI_DA850=y
 # CONFIG_DA8XX_MAX_SPEED_372 is not set
 # CONFIG_DA8XX_MAX_SPEED_408 is not set
 CONFIG_DA8XX_MAX_SPEED_456=y
+CONFIG_DAVINCI_UART1_AFE=y
 CONFIG_ARCH_DAVINCI_DA8XX=y
 # CONFIG_ARCH_DAVINCI_DM365 is not set
 
@@ -250,6 +251,9 @@ CONFIG_ARCH_DAVINCI_DA8XX=y
 # DaVinci Board Type
 #
 CONFIG_MACH_DAVINCI_DA850_EVM=y
+CONFIG_WIFI_CONTROL_FUNC=y
+# CONFIG_DA850_USE_MMC0 is not set
+CONFIG_DA850_USE_MMC1=y
 CONFIG_DA850_UI_NONE=y
 # CONFIG_DA850_UI_RMII is not set
 # CONFIG_DA850_UI_CLCD is not set
@@ -507,10 +511,39 @@ CONFIG_NETFILTER_ADVANCED=y
 # CONFIG_HAMRADIO is not set
 # CONFIG_CAN is not set
 # CONFIG_IRDA is not set
-# CONFIG_BT is not set
+CONFIG_BT=y
+CONFIG_BT_L2CAP=y
+CONFIG_BT_SCO=y
+CONFIG_BT_RFCOMM=y
+CONFIG_BT_RFCOMM_TTY=y
+CONFIG_BT_BNEP=y
+CONFIG_BT_BNEP_MC_FILTER=y
+CONFIG_BT_BNEP_PROTO_FILTER=y
+CONFIG_BT_HIDP=y
+
+#
+# Bluetooth device drivers
+#
+# CONFIG_BT_HCIBTUSB is not set
+# CONFIG_BT_HCIBTSDIO is not set
+CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART_H4=y
+CONFIG_BT_HCIUART_BCSP=y
+CONFIG_BT_HCIUART_LL=y
+# CONFIG_BT_HCIBCM203X is not set
+# CONFIG_BT_HCIBPA10X is not set
+# CONFIG_BT_HCIBFUSB is not set
+# CONFIG_BT_HCIVHCI is not set
+# CONFIG_BT_MRVL is not set
 # CONFIG_AF_RXRPC is not set
 CONFIG_WIRELESS=y
+CONFIG_WIRELESS_EXT=y
+CONFIG_WEXT_CORE=y
+CONFIG_WEXT_PROC=y
+CONFIG_WEXT_SPY=y
+CONFIG_WEXT_PRIV=y
 # CONFIG_CFG80211 is not set
+CONFIG_WIRELESS_EXT_SYSFS=y
 # CONFIG_LIB80211 is not set
 
 #
@@ -750,6 +783,7 @@ CONFIG_TI_DAVINCI_EMAC=y
 # CONFIG_NETDEV_1000 is not set
 # CONFIG_NETDEV_10000 is not set
 CONFIG_WLAN=y
+CONFIG_WL1271_WLAN=y
 # CONFIG_USB_ZD1201 is not set
 # CONFIG_HOSTAP is not set
 
@@ -1860,7 +1894,7 @@ CONFIG_CRYPTO=y
 CONFIG_BITREVERSE=y
 CONFIG_GENERIC_FIND_LAST_BIT=y
 CONFIG_CRC_CCITT=m
-# CONFIG_CRC16 is not set
+CONFIG_CRC16=y
 CONFIG_CRC_T10DIF=m
 # CONFIG_CRC_ITU_T is not set
 CONFIG_CRC32=y
-- 
1.7.0.4

